.changeset/breezy-cycles-count.md

```diff
@@ -0,0 +1,42 @@
+---
+'@backstage/plugin-events-backend-test-utils': patch
+'@backstage/plugin-events-backend': patch
+'@backstage/plugin-events-node': patch
+---
+
+Add new `EventService` as well as `eventServiceRef`/`eventServiceFactory` for the new backend system.
+
+**Summary:**
+
+- new:
+  `EventService`, `eventServiceRef`, `eventServiceFactory`, `TestEventService`
+- deprecated:
+  `EventBroker`, `EventPublisher`, `EventSubscriber`, `DefaultEventBroker`, `EventsBackend`,
+  most parts of `EventsExtensionPoint` (alpha),
+  `TestEventBroker`, `TestEventPublisher`, `TestEventSubscriber`
+
+Add the following to your backend _(new backend system)_ to add the `eventServiceFactory` for `eventServiceRef`:
+
+```ts
+backend.add(import('@backstage/plugin-events-node'));
+```
```

Comment:
Let's remove the need for this. Either by shipping it as default or by making sure duplicate instances can still communicate with each other.

Reply:
What would you suggest to achieve this? Would you add this to the core services instead?